.github/workflows/unit-test.yml

```diff
@@ -72,9 +72,15 @@ jobs:
                     - '8.1'
                     - '8.2'
                 multisite: [false, true]
+                wordpress: [''] # Latest WordPress version.
+                include:
+                    # Test with the previous WP version.
+                    - php: '7.4'
```

Comment:
Is there a reason to test the previous WP release with PHP 7.4 only?
Gutenberg and WordPress should support identical PHP versions, i.e., 5.6 through 8.2.

Reply:
Yeah, I didn't want to use up too many CI resources (I think we have a limit to the number of jobs we can run in parallel?), and doubling the number of unit tests would add quite significantly to it (I know we fail eagerly, but we still have all those PRs where _all_ tests pass 😬).

The class of failures that we won't detect with this PR are failures with the previous WP version on any of the other (i.e. non-7.4) PHP versions. But that intersection seems very small to me, since it'd mean we're dealing with some WP feature that somehow works on PHP 7.4 but doesn't on one of the other ones; yet tests pass for _all_ PHP versions when run with the current WP version.

IMO this is so unlikely that we can go with only one PHP version for now (which already gives us quite the confidence boost in unit tests compared to what we have now). If we ever end up with a scenario as described above where a bug goes undetected, we can revisit and increase coverage to span other PHP versions 😊

Reply:
> previous WP version on any of the other (i.e. non-7.4) PHP versions

We actually did run into this issue a few times in the past. Mostly because PHP 8 turns several warnings into errors. So code that worked in 7.4 started creating errors in 8.0. We ran into this issue at least once also using the previous WordPress version 😬 

But yeah, CI space is valuable... But I wonder if we could make the unit test suites faster. 6 minutes for phpunit feels like a long time 🤔

Reply:
Could do one 7.x and one 8.x version maybe? Would that about cover it?

Reply:
I think PHP 8 plus the oldest WP version we support would cover the issues we found 👍